examples/long-paths.js

```diff
@@ -0,0 +1,41 @@
+// This test creates a file with a path that's over 300 characters
```

Comment:
Thanks for putting this together, and for including the useful and informative comment.